test/tests/parser/active_inactive/tests

```diff
@@ -0,0 +1,29 @@
+[Tests]
+  [./inactive_active]
+    type = Exodiff
+    input = 'active_inactive.i'
+    exodiff = 'active_inactive_out.e'
+    cli_args = 'Outputs/exodus=true'
+  [../]
+
+  [./inactive_section_missing]
+    type = RunException
+    input = 'active_inactive.i'
+    cli_args = 'AuxKernels/active=aux5'
+    expect_err = 'One or more active/inactive lists in the input file are missing a referenced section'
+  [../]
+
+  [./active_section_missing]
+    type = RunException
+    input = 'active_inactive.i'
+    cli_args = 'Kernels/inactive=convection BCs/inactive=top'
```

Comment:
Yes, however the parser has to parse the whole file since Getpot allows you to put sections anywhere so these errors are saved up. I put two here so that we can see that the parser shows you all sections that have problems, not just the first one it encounters.